CHANGELOG.md

```diff
@@ -11,10 +11,10 @@
 
 * The Mapbox Navigation SDK for iOS license has changed from the ISC License to the Mapbox Terms of Service. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * You can now install MapboxNavigation using Swift Package Manager, but you can no longer install it using Carthage. If you previously installed MapboxNavigation using Carthage, use Swift Package Manager instead. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
-* MapboxNavigation now depends on [MapboxMaps v10.0.0-rc.7](https://github.com/mapbox/mapbox-maps-ios/). ([#3297](https://github.com/mapbox/mapbox-navigation-ios/pull/3297))
-* MapboxNavigation now depends on MapboxNavigationNative v65.0.1. ([#3320](https://github.com/mapbox/mapbox-navigation-ios/pull/3320))
-* MapboxNavigation now depends on MapboxCommon v17.1.0. ([#3297](https://github.com/mapbox/mapbox-navigation-ios/pull/3297))
 * MapboxNavigation now depends on MapboxMobileEvents v1.0.3. ([#3320](https://github.com/mapbox/mapbox-navigation-ios/pull/3320))
+* MapboxNavigation now depends on [MapboxMaps v10.0.0-rc.8](https://github.com/mapbox/mapbox-maps-ios/). ([#3342](https://github.com/mapbox/mapbox-navigation-ios/pull/3342))
+* MapboxNavigation now depends on MapboxNavigationNative v66.0.0. ([#3342](https://github.com/mapbox/mapbox-navigation-ios/pull/3342))
```

Comment:
Were there any changes in MapboxNavigationNative v66.0.0 worth mentioning as improvements over the version used by navigation SDK v1._x_?

Reply:
Yes, there were several deprecations in `v66.0.0`:
- `Navigator.getStatus()` was deprecated in favor of `NavigatorObserver`.

Reply:
`MapboxNavigationNative.Navigator` is fully encapsulated by `MapboxCoreNavigation.Navigator`, which itself is internal, so we can leave that out of the changelog. I was wondering if there were any behavioral changes worth mentioning, but I guess this release was relatively uneventful on that front.